.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+# Omakase Ruby styling for Rails
+inherit_gem: { rubocop-rails-omakase: rubocop.yml }
```

Comment:
ditto

Reply:
Bhai, added a new line at the bottom of the file.

Reply:
Bhai, added a new line at the bottom of the file.